### PR TITLE
frontend/config: Enabled Yatri Coins and Monthly Leaderboard in YS

### PR DESCRIPTION
**Pull request by slow-codex**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Enabled Coins feature in Durgapur and Asansol.

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Review by max-keviv — APPROVED** [review]

**Review by ashishsingh101 — APPROVED** [review]

File: Frontend/android-native/app/src/driver/jatriSaathiDriver/assets/juspay/juspay_assets.json
Comment (max-keviv) [inline Frontend/android-native/app/src/driver/jatriSaathiDriver/assets/juspay/juspay_assets.json:1]: why this changes though ?

**Review by max-keviv — APPROVED** [review]

File: Frontend/android-native/app/src/driver/jatriSaathiDriver/assets/juspay/juspay_assets.json
Comment (slow-codex) [reply Frontend/android-native/app/src/driver/jatriSaathiDriver/assets/juspay/juspay_assets.json:1]: Done by `driverJuspayAssets.sh` during apk generation [here](https://github.com/nammayatri/nammayatri/blob/main/Frontend/driverJuspayAssets.sh)
